web/frontend/src/components/camera/live-camera.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang='ts'>
+import { useConnect } from '@/hooks/robot-client';
+import type { CameraManager } from './camera-manager';
+
+export let cameraName: string;
+export let cameraManager: CameraManager
+
+let videoEl: HTMLVideoElement;
+
+useConnect(() => {
+  videoEl.srcObject = cameraManager.videoStream;
+  cameraManager.onOpen = () => {
+    videoEl.srcObject = cameraManager.videoStream;
+  };
+
+  cameraManager.addStream();
+
+  return () => {
+    cameraManager.onOpen = undefined;
+    cameraManager.removeStream();
+  }
+});
+
+</script>
+
+<video
+  bind:this={videoEl}
+  muted
+  autoplay
+  controls={false}
+  playsinline
+  aria-label={`${cameraName} stream`}
+  class="clear-both h-fit transition-all duration-300 ease-in-out"
+/>
```

Comment:
[Nit] missing newline

Reply:
Had planned on adding prettier in a follow up, any issues with for waiting for that?